web/src/stream_list.js

```diff
@@ -633,9 +633,13 @@ export function set_event_handlers() {
         }
     }
 
+    const $stream_scroll_container = $("#left_sidebar_scroll_container");
     // check for user scrolls on streams list for first time
-    ui.get_scroll_element($("#left_sidebar_scroll_container")).on("scroll", () => {
+    ui.get_scroll_element($stream_scroll_container).on("wheel", () => {
         has_scrolled = true;
+    });
```

Comment:
Does scrolling on a trackpad use the `wheel` event? I worry this change will break the logic for trackpads, but don't know how those work and MDN didn't answer this clearly for me in a quick look.